bitnami/kafka/templates/_helpers.tpl

```diff
@@ -405,11 +406,27 @@ kafka: auth.clientProtocol auth.externalClientProtocol auth.interBrokerProtocol
 
 {{/* Validate values of Kafka - number of replicas must be the same as NodePort list */}}
 {{- define "kafka.validateValues.nodePortListLength" -}}
-{{- $replicaCount := int .Values.replicaCount }}
-{{- $nodePortListLength := len .Values.externalAccess.service.nodePorts }}
-{{- if and .Values.externalAccess.enabled (not .Values.externalAccess.autoDiscovery.enabled) (not (eq $replicaCount $nodePortListLength )) (eq .Values.externalAccess.service.type "NodePort") -}}
+{{- $replicaCount := int .Values.replicaCount -}}
+{{- $nodePortListLength := len .Values.externalAccess.service.nodePorts -}}
+{{- $nodePortLe := empty .Values.externalAccess.service.nodePorts -}}
+{{- $nodePortLr := eq $nodePortListLength $replicaCount -}}
+{{- $externalIPLe := empty .Values.externalAccess.service.externalIPs -}}
+{{- if and .Values.externalAccess.enabled (not .Values.externalAccess.autoDiscovery.enabled) (eq .Values.externalAccess.service.type "NodePort") (or (and (not $nodePortLe) (not $nodePortLr)) (and $nodePortLe $externalIPLe)) -}}
```

Comment:
Hi, could you use full lenght variable here ? I think it would be clear if you use something like `$isNodePortListempty` or something similar.
Same for the others abreviations in those lines.

Reply:
I made them more "speaking".